### ISSUE TITLE
Error: Channel not found: C

### ISSUE THREAD

**Issue by dtrax** [janeasystems/nodejs-mobile#264]
* **Project**: NodeJS Cordova plugin
* **Version**: Release 0.4.0
* **Mobile device**: Android - Oneplus 7
* **Mobile OS and version**: Android 9
* **Development Node.js**: 
* **Development platform**: Cordova 9.0.0 / Cordova-android 8.1.0

I have edited the plugin to work with my cordova version by replacing all the requireCordovaModule with simple require in hooks JS. So it installs perfectly and app build is successful.
However, I get this error in the console when I run the app, and the channel communication doesn't work. Is it something I can fix or is it an issue with the versions that I could not fix. Please let me know, Thanks a lot.

```
Error: Channel not found: C
allChannelsListener | @ | plugins/nodejs-mobil…/nodejs_apis.js:101
```

**Comment by jaimecbernardo** [comment]
Hi,
Haven't ever seen this error before.

Here's how other users have been able to use the plugin on cordova-android 8.1.0:
https://github.com/JaneaSystems/nodejs-mobile/issues/239

**Comment by dtrax** [comment]
Hey there.
Yes I have made the changes. No issues with the build process. I digged down more and I have found an underlying issue.

So it's this error in the console. And still the NodeJS variable is defined.
`Error: Channel not found: C
allChannelsListener | @ | plugins/nodejs-mobil…/nodejs_apis.js:101`

When I hit nodejs.startEngine, there is an error from cordova.exec function in nodejs_api.js - 
It says,
`Class not found`

That's it. I can't detect which class is it talking about. I've checked and no, the file is not missing. It's throwing the error before trying to access the script file.
